### ISSUE TITLE
Database management tool: step 1.

### ISSUE THREAD

**Issue by nemerle** [Segs/Segs#88]
A simple tool that will create sqlite databases based on the sql template fiiles:
Basic logic should look like
```cpp
...
if(weAreNotRunningFromServerDir())
    return RUN_US_FROM_SERVER_DIR;
if(databaseFilesAreHere())
    return NOTHING_TO_DO;
// create auth server account db
QString path = SEGS_ACCOUNTS_DB_PATH;
QSqlDatabase db = QSqlDatabase::addDatabase("QSQLITE");
db.setDatabaseName(path);
db.open();
QSqlQuery query;
QFile db_template("segs.sql");
if(db_template.open(QFile::ReadOnly)
    query.exec(db_template.ReadAll());
db.close();
// create game server character db
...
```

**Comment by broxen** [comment]
Should I be using qDebug() instead of my own qout function for displayed output?

I thought qDebug was for stderr output only.

**Comment by nemerle** [comment]
qDebug is redirectable using qInstallMsgHandler

**Comment by broxen** [comment]
This appears to be resolved as of #129

**Comment by nemerle** [comment]
Indeed it is :)
